### DISCUSSION TITLE
🔗 Templates: Better export/duplication (Duplicate)

### DISCUSSION THREAD

**Discussion by bkjproductions** [elementor/elementor#14166] [Feature Request]
When exporting a Template from Elementor, it's great that it name-dates the file. 
Example: **elementor-23-2021-01-25.json**
However, it would be just as simple to name the file "My-Great-Template-2021-01-25.json" if the template were named "My Great Template."

However, one also has to question, if one of our goals is to simply **duplicate** a Template, why do we have to export it at all? Why not have a "Clone" button there in the Templates list? (Nice new interface, BTW, go ahead and make it default and bypass the WP-style list.)
Currently trying to use any of the various WP plugins for duplicating Posts does not work; they all leave off some critical meta value that prevents the duplicated Template from being used.

![add-clone](https://user-images.githubusercontent.com/4551281/110343421-7b2c8980-7ffa-11eb-80c7-ae3f8f58bc51.jpg)

**Comment by bkjproductions** [comment]
**And, another thing:** When I am at this panel, currently there is no way to IMPORT a template (that I can readily see) --- just exporting? So do I have to go back to the WordPress standard "posts" list like you would see at  /wp-admin/edit.php?post_type=elementor_library&tabs_group=theme
I thought you were trying to get people to use "the New Theme Builder: More Intuitive and Visual Than Ever" --- so it's not that intuitive, is what I am saying. :)

**Reply by bkjproductions** [reply]
Looks like they have fixed the missing import-template button within the Theme Builder.
BUT, the icon is pointing DOWN rather than UP. I think it should point up.
![should be upload icon](https://user-images.githubusercontent.com/4551281/145690061-509c454d-1421-4fdf-afd6-3dd9f4054ef8.JPG)

**Comment by nicholaszein** [comment]
Hello there @bkjproductions!

thanks for submitting your feature request! 🙏 

We are in the process of organizing all Feature Requests and consolidating everything in one place. Part of this process is to individualize each request so we can track the community's interest better, and implement features as they are voted.

Unfortunately, I see you are requesting two different features: 

1. An option to include the sanitized template name in the template file so it's easier to identify it when we export the template from the Theme Builder (this is a nice one! 😉);
2. An option to include a button to Duplicate or Clone a Theme Builder template directly from the Theme Builder interface.

### 🗃️ Please understand that while we do welcome new feature requests, we can't track multiple requests per thread. 

Please follow our guidelines for submitting a new [🚀 Feature Request](https://elemn.to/gh-new-feature-request), and submit only ☝️ **_<ins>one request per thread</ins>_** so we can track them better and allow users to vote only for the features they want to see implemented.

I'm closing this request as a duplicate. I hope you can understand.

Kind regards
